lms/djangoapps/lms_xblock/runtime.py

```diff
@@ -72,6 +71,64 @@ def local_resource_url(block, uri):
     return xblock_local_resource_url(block, uri)
 
 
+def lms_wrappers_aside(block, aside, view, frag, context, request_token=None):
+    """
+    Creates a div which identifies the aside, points to the original block,
+    and writes out the json_init_args into a script tag.
+
+    The default implementation creates a frag to wraps frag w/ a div identifying the xblock. If you have
+    javascript, you'll need to override this impl
+    """
+    if not frag.content:
+        return frag
+
+    runtime_class = 'LmsRuntime'
+    extra_data = {
+        'block-id': quote_slashes(str(block.scope_ids.usage_id)),
+        'course-id': quote_slashes(str(block.course_id)),
```

Comment:
```suggestion
        'course-id': quote_slashes(str(block.scope_ids.usage_id.context_key)),
```

We have already deprecated `course_id`.